pkg/providers/v1/aws.go

```diff
@@ -1156,6 +1164,13 @@ func (s *awsSdkEC2) DescribeSubnets(request *ec2.DescribeSubnetsInput) ([]*ec2.S
 	return response.Subnets, nil
 }
 
+func (s *awsSdkEC2) DescribeAvailabilityZones(request *ec2.DescribeAvailabilityZonesInput) ([]*ec2.AvailabilityZone, error) {
+	// AZs are not paged
+	response, err := s.ec2.DescribeAvailabilityZones(request)
+	// do not wrap the error for being able to check its type down the road
```

Comment:
If you use `fmt.Errorf` with a `%w` format string then in later calls `errors.Is` can check this by unwrapping